### ISSUE TITLE
NMS.KK.OBD.000 The outbound-dialer module shall create a daily file of outgoing calls

### ISSUE THREAD

**Issue by frankhuster** [motech-implementations/mim#197]
4.3.1 TargetFile Notification API
NMS shall invoke this API to notify IVR platform when a target file is ready.

**Comment by llavoie** [comment]
I assume you didn't mean for this to be auto-closed yet. :)

**Comment by frankhuster** [comment]
Ha, I wish...

Thanks.

**Comment by larubbio** [comment]
I'm going to close this without updating the SRS spec.  We do have an API for this, and it isn't a req. that we create the file.  It is a req. that calls are placed.  The mechanism of communication between MOTECH and the IVR are an internal detail.

**Comment by frankhuster** [comment]
Waiting to hear how/if we specify which content to play for each call. Right now the spec says:
```
        /*
         * #7 ContentFileName
         *
         * Content file to be played
         */
```

**Comment by frankhuster** [comment]
But I'm not sure how to generate this from what we have in the database. @larubbio suggested we only put the weekId (and something identifying which message of the week) instead of a filename

**Comment by frankhuster** [comment]
Fixed by #246
